docs/source/tutorial-arm-ethos-u.md

```diff
@@ -0,0 +1,215 @@
+# Arm&reg; Ethos&trade;-U NPU Backend Tutorial
```

Comment:
`&reg` and `&trade` in title feels odd - too corporate :p  
We can put it somewhere on the page that Arm is a registered trademark etc.. 

I am not a lawyer :P